docs/scripts/hide-sidebar.js

```diff
@@ -3,4 +3,14 @@ window.onload = function() {
   if (sideBar) {
     sideBar.style.display = 'none';
   }
+
+  const docsLinks = [].slice.call(window.document.getElementsByClassName('md-nav__link'));
+  const apiLinks = docsLinks.filter(x => {
+    const text = x.innerText.trim();
+    return text === 'API V2' || text === 'API Import V2';
+  });
+
+  for (let link of apiLinks) {
```

Comment:
rather than having another loop, could you set the attribute as you loop through the `docsLinks` above? Reduce the O factor here some